pkg/limits/config.go

```diff
@@ -93,6 +96,18 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 		DefaultNumPartitions,
 		"The number of partitions for the Kafka topic used to read and write stream metadata. It is fixed, not a maximum.",
 	)
+	f.StringVar(
+		&cfg.ConsumerGroup,
+		"ingest-limits.consumer-group",
+		DefaultConsumerGroup,
+		"The consumer group for the Kafka topic used to read stream metadata records.",
+	)
+	f.StringVar(
+		&cfg.Topic,
+		"ingest-limits.topic",
+		"",
```

Comment:
We should set a default here?

Reply:
I kept this as we do in `kafka_config.topic` to make this an explicit user choice.